docs/source/api/v5/types.rst

```diff
@@ -152,7 +152,7 @@ Response Structure
 		"response": [
 		{
 			"id": 3004,
-			"lastUpdated": "2020-02-26 18:58:41+00",
+			"lastUpdated": "2020-02-26T20:49:28.111111Z"",
```

Comment:
missing time offset, like `+05:30`. This is how RFC format looks like: `2023-05-25T15:59:33.7096-06:00`